source/Calamari.Tests/KubernetesFixtures/KubernetesContextScriptWrapperLiveFixtureBase.cs

```diff
@@ -184,17 +183,6 @@ protected void DoDiscovery(AwsAuthenticationDetails authenticationDetails)
                                        .Argument("extensions", string.Join(',', extensions)));
             }
         }
-
-        protected class ServiceMessageCollectorLog : InMemoryLog
-        {
-            public List<ServiceMessage> ServiceMessages { get; } = new List<ServiceMessage>();
-            public override void WriteServiceMessage(ServiceMessage serviceMessage)
-            {
-                ServiceMessages.Add(serviceMessage);
-                
-                base.WriteServiceMessage(serviceMessage);
-            }
-        }
```

Comment:
I incorporated this into `InMemoryLog` - didn't see why we wouldn't do so since `InMemoryLog` is only used in tests.

Reply:
This looks like a good change.
To keep your PR clean and focused though, I'd recommend this change, be made in its own PR.
The smaller the set of changes the easier it is for the reviewer to understand the full context